error-monitoring/test/bot.test.ts

```diff
@@ -95,6 +87,14 @@ describe('ErrorIssueBot', () => {
       .mockReturnValue('issue body');
   });
 
+  afterEach(() => {
+    // Fail the test if there were unused nocks.
+    if (!nock.isDone()) {
+      throw new Error('Not all nock interceptors were used!');
+    }
+    nock.cleanAll();
+  });
+
```

Comment:
Does this change do anything?

Reply:
Yeah, it moves the `afterEach` outside of the `beforeEach` - starting in Jest 27 nesting these became a fatal error

Reply:
The `afterEach` was..._inside_ the `beforeEach`? 🙀

Reply:
Screaming cat emoji indeed 😂